ceph/README.md

```diff
@@ -0,0 +1,45 @@
+# Ceph Object Storage
+
+[Ceph](https://ceph.io) provides S3 compatible object storage via the RADOS GateWay(RGW) Service
+
+Deploys [cn-core](https://github.com/cn-core) image that provides minimal object storage for small deployments based on a release of [ceph-container](https://github.com/ceph/ceph-container).
+
+***NOTE***: This container MUST run as a PRIVILEGED container with anyuid enabled
+
```

Comment:
Is there a way around this?

Reply:
I removed the Privilege execution. It only requires anyuid permissions now